docs/modules/ROOT/pages/custom-attributes.adoc

```diff
@@ -0,0 +1,75 @@
+= Custom Attributes
+
+Custom Attributes are a string of space-separated `_attribute_=_value_` pairs added to the header of each AsciiDoc page viewed by the extension before it is rendered.
+
+Custom Attributes allow you to use the features of AsciiDoc in documents without having to write them each time, or include a separate file of attribute settings.
```

Comment:
Is "Custom Attributes" referring to the feature? If so, I think it's a bit misleading. We could say that we can declare/define attributes using the "Custom attributes" field but "Custom attributes" is also the name of non-built-in attributes in the documentation (so it might be confusing)

Reply:
https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes/#what-are-document-attributes is a good reference

Reply:
Agree. I'll rework the wording to remove confusion.

In this file, it is specifically the "Extension Custom Attributes option field". Will update...

The official docs refer to  "Built-in" and "User defined" attributes not custom.